README.md

```diff
@@ -1,3 +1,56 @@
 # perspectiveapi-js-client
 
-A simple example JS/TS client library for the [perspective api](https://perspectiveapi.com/)
+A simple example JS/TS client library for the [Perspective API](https://www.perspectiveapi.com/).
+This client accepts text and returns some of the text's attributes.
+
+## Usage
+
+First, you'll need an API key from ConversationAI's [Perspective API](https://www.perspectiveapi.com/).
+
+```
+require { Client } from "perspectiveapi-js-client";
+const client = new Client(YOUR_AP_KEY);
+const results = client.getScores(text, options);
+
+```
+
+## Methods
+
+This client exposes one method, getScores.
+
+`getScores(text, options);`
+
+### Parameters
+
+`text` *(Required)*
+
+A string between 1 and 3000 characters representing a piece of text (e.g. a comment on an article) to be analyzed by Perspective API.
+
+`options`
+
+An optional object that specifies options for the API.
+
+Accepted keys in the options object are:
+
+Field                    | Description
+-----                    | -----------
+`context.entries`        | A list of objects providing the context for `text`.
+`context.entries[].text` | The text of a context object.
+`attributes`             | An array of strings specifying the attributes to get scores for. See the [Perspective docs](https://github.com/conversationai/perspectiveapi/blob/master/api_reference.md#models) for a list of available attributes. Default: `["SPAM", "TOXICITY"]`
+`languages`              | A list of [ISO 631-1](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes) two-letter language codes specifying the language(s) that `text` is in (for example, "en", "es", "fr", etc). Default: `["EN"]`
+`doNotStore`             | Whether the API is permitted to store `text` and `context` from this request. Default: `true`
```

Comment:
@iislucas It sure could be. Can you imagine a use case for that? Doesn't Perspective always do html stripping on its side?